app/Content/Content.js

```diff
@@ -3,7 +3,7 @@ import {buildList, showFavorties, addArticles} from './actions'
 import React, {Component} from 'react';
```

Comment:
Might be good to add some spacing within the brackets to be consistent and match line 6